Fixed TracingKafkaClientSupplier documentation (#7434)
strimzi/strimzi-kafka-operator: documentation/modules/tracing/proc-instrumenting-kafka-streams-with-tracers.adoc

+
You enable instrumentation in Kafka Streams applications by adding the tracing JARs as dependencies to your project.

> * To instrument Kafka Streams with OpenTelemetry, you'll need to write a custom `TracingKafkaClientSupplier`.
> * The custom `TracingKafkaClientSupplier` can extend the Kafka provided `DefaultKafkaClientSupplier` but overriding the producer and consumer creation methods in order to wrap the instances with the telemetry related code.

Reviewer: ```suggestion
* To instrument Kafka Streams with OpenTelemetry, you'll need to write a custom `TracingKafkaClientSupplier`.
+
The custom `TracingKafkaClientSupplier` can extend Kafka's `DefaultKafkaClientSupplier`, overriding the producer and consumer creation methods to wrap the instances with the telemetry-related code.
```
